Integrations/integration-TCell.yml

```diff
@@ -0,0 +1,635 @@
+            Contents: respBody,
+            ContentsFormat: formats.json,
+            HumanReadable: tableToMarkdown('TCell App Configs For: ' + args.appid, [respBody.config]),
+            EntryContext: {'tcell_app_config_latest': [respBody.config]}
+        };
+    }
+    var tcellAppConfigData = function() {
+        var urlPath = urlDictionary['tcell-app-config-data'].replace('%APP_ID%', args.appid).replace('%CONFIG_ID%', args.configid);
+        var respBody = sendRequest(methodDictionary['tcell-app-config-data'],urlPath,'', '');
+        return {
+            Type: entryTypes.note,
+            Contents: respBody,
+            ContentsFormat: formats.json,
+            HumanReadable: tableToMarkdown('TCell App Configs For: ' + args.appid, [respBody.config]),
+            EntryContext: {'tcell_app_config_data': [respBody.config]}
+        };
+    }
+    var tcellAppPushConfig = function() {
+        var urlPath =  urlDictionary['tcell-app-push-config'].replace('%APP_ID%', args.appid);
+        var blacklistips = [];
+        var whitelistips = [];
+        if('blacklistips' in args) {
+            blacklistips = args.blacklistips.replace(' ','').split(',');
+        }
+        if('whitelistips' in args) {
+            whitelistips = args.whitelistips.replace(' ','').split(',');
+        }
+        var configBody = {
+            "version": 1,
+            "blockedips": {
+              "suspiciousMode": "Filtered",
+              "blacklistedIps": blacklistips,
+              "whitelistedIps": whitelistips,
+              "version": 1
+            }
+          };
+
+        var respBody = sendRequest(methodDictionary['tcell-app-push-config'], urlPath,'',configBody, '','');
+        var message = ('id' in respBody)? respBody.id : 'Nothing to update';
+        return {
+            Type: entryTypes.note,
+            Contents: respBody,
+            ContentsFormat: formats.json,
+            HumanReadable: '# Successfully Pushed The TCell Config - #' + message,
+            EntryContext: {'tcell_app_config_update': [respBody]}
+        };
+    }
+
+
+
+    var sendRequest = function(method, urlpath, headers, bodyData, cookies) {
+            var completeUrl = server + '/' + urlpath;
+            var reqObj = {};
+            reqObj.Method = method;
+            reqObj.Headers = {
+                        'Authorization': ['Bearer ' + apiKey],
+                        'Content-Type': ['application/json']
+                    };
+            if(bodyData) {
+                reqObj.Body = JSON.stringify(bodyData);
+            }
+            var res = http(
+                completeUrl,
+                reqObj,
+                params.insecure,
+                proxy,
+                true
+            );
+            if (res.StatusCode < 200 || res.StatusCode >= 300) {
+                throw 'Action failed, request status code: ' + res.StatusCode + ' and Body: ' + res.Body + '.';
+            }
+
+            if (res.Headers['Content-T
```

Comment:
That is the difficult portion about this at the moment. Determining what is important for the context and fully describing it is difficult as it is an Alpha version and their API is also being updated so this may actually change in a later version.